EPRN.PRNS.API/Controllers/PRNController.cs

```diff
@@ -20,12 +20,13 @@ public PRNController(IPrnService prnService)
         }
 
         [HttpPost]
-        [Route("/api/[controller]/Create/Category/{category}/Material/{materialId}")]
+        [Route("/api/[controller]/Create/Category/{category}/Material/{materialId}/{userReferenceId}")]
```

Comment:
As a string parameter this should probably be supplied in the body of the request. It doesn't represent anything from a RESTful sense